Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -8,41 +8,44 @@ import Foundation
 import XCTest
 import PathKit
 
-private let colorCode: (String) -> String = ProcessInfo().environment["XcodeColors"] == "YES" ? { "\u{001b}[\($0);" } : { _ in "" }
+private let colorCode: (String) -> String =
+  ProcessInfo().environment["XcodeColors"] == "YES" ? { "\u{001b}[\($0);" } : { _ in "" }
 private let (msgColor, reset) = (colorCode("fg250,0,0"), colorCode(""))
-private let okCode = (num: colorCode("fg127,127,127"), code: colorCode(""))
-private let koCode = (num: colorCode("fg127,127,127") + colorCode("bg127,0,0"), code: colorCode("fg250,250,250") + colorCode("bg127,0,0"))
+private let okCode = (num: colorCode("fg127,127,127"),
+                      code: colorCode(""))
+private let koCode = (num: colorCode("fg127,127,127") + colorCode("bg127,0,0"),
+                      code: colorCode("fg250,250,250") + colorCode("bg127,0,0"))
 
 func diff(_ result: [String: Any], _ expected: [String: Any], path: String = "") -> String? {
-  
+
   // check keys
   if Set(result.keys) != Set(expected.keys) {
-    let lhs = result.keys.map{" - \($0): \(result[$0] ?? "")"}.joined(separator: "\n")
-    let rhs = expected.keys.map{" - \($0): \(expected[$0] ?? "")"}.joined(separator: "\n")
+    let lhs = result.keys.map { " - \($0): \(result[$0] ?? "")" }.joined(separator: "\n")
+    let rhs = expected.keys.map { " - \($0): \(expected[$0] ?? "")" }.joined(separator: "\n")
     let path = (path != "") ? " at '\(path)'" : ""
     return "\(msgColor)Keys do not match\(path):\(reset)\n>>>>>> result\n\(lhs)\n======\n\(rhs)\n<<<<<< expected"
   }
-  
+
   // check values
   for (key, lhs) in result {
     guard let rhs = expected[key] else { continue }
-    
+
     if let error = compare(lhs, rhs, key: key, path: path) {
       return error
     }
   }
-  
+
   return nil
 }
 
 func compare(_ lhs: Any, _ rhs: Any, key: String, path: String) -> String? {
   let keyPath = (path == "") ? key : "\(path).\(key)"
-  
-  if let lhs = lhs as? Bool, let rhs = rhs as? Bool, lhs == rhs { return nil}
-  else if let lhs = lhs as? Int, let rhs = rhs as? Int, lhs == rhs { return nil }
-  else if let lhs = lhs as? Float, let rhs = rhs as? Float, lhs == rhs { return nil }
-  else if let lhs = lhs as? String, let rhs = rhs as? String, lhs == rhs { return nil }
-  else if let lhs = lhs as? [Any], let rhs = rhs as? [Any], lhs.count == rhs.count {
+
+  if let lhs = lhs as? Bool, let rhs = rhs as? Bool, lhs == rhs { return nil
```

Comment:
If we're going to go to the next line for the closing brace (probably because SwiftLint wants us to right?) I'd prefer to go to the next line after the opening brace too to be consistent.